pkg/util/oc/pod_commands.go

```diff
@@ -125,10 +125,10 @@ func (o OC) WaitCondition(t test.TestHelper, ns string, kind string, name string
 	t.T().Helper()
 	retry.UntilSuccessWithOptions(t, retry.Options().MaxAttempts(30), func(t test.TestHelper) {
 		shell.Execute(t,
-			fmt.Sprintf(`oc wait -n %s %s/%s --for condition=%s  --timeout %s`, ns, kind, name, condition, "10s"),
+			fmt.Sprintf(`oc wait -n %s %s/%s --for condition=%s  --timeout %s`, ns, kind, name, condition, "20s"),
 			assert.OutputContains(condition,
-				fmt.Sprintf("Condition %s met by %s %s/%s", condition, kind, ns, name),
-				fmt.Sprintf("Condition %s not met %s %s/%s, retrying", condition, kind, ns, name)))
+				fmt.Sprintf("Condition %s met by %s %s/%s", condition, ns, kind, name),
+				fmt.Sprintf("Condition %s not met %s %s/%s, retrying", condition, ns, kind, name)))
```

Comment:
What was wrong with the previous message? We use the `<namespace>/<name>` format in several places.

Reply:
Whit the previous message I think it's a bit confusing because we said: `Condition Ready not met smcp istio-system/default` And the command for the wait is `oc wait -n istio-system smcp/basic ...`. If you read the message in this case the one who reads can think that the command is being appliying like `oc wait -n smcp istio-system/default`. What do you think, or maybe we can change the message for something like: `Condition xxxxx not met by kind: xxxx in namespace/name`

Reply:
Will change this